CHANGES.md

```diff
@@ -4,7 +4,12 @@ New Grammars:
 
 - added 3rd party Lang grammar to SUPPORTED_LANGUAGES [AdamRaichu][]
 
+Grammars:
+
+- Added support for highlighting GraphQL tagged template strings in JS/TS code [Ali Ukani][]
+
 [AdamRaichu]: https://github.com/AdamRaichu
+[Ali Ukani]: https://github.com/ali
 
 ## Version 11.8.0
```

Comment:
I noticed that the latest published release is v11.7.0 (instead of v11.8.0), but I added my entry to the latest section (for v11.9.0) anyway.

Reply:
Hmmm, maybe changelog got messed up... I'll look into it.